installations/InstallationsSnippets/AppDelegate.swift

```diff
@@ -38,6 +38,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
     // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
   }
+    
+  func handleInstallationIDChange() {
+    // [START handle_installation_id_change]
+    NotificationCenter.default.addObserver(forName: .InstallationIDDidChange,
```

Comment:
The method returns an object required to unsubscribe from the notifications (see the [docs](https://developer.apple.com/documentation/foundation/notificationcenter/1411723-addobserver)). We should add it to the snippet to follow best practices.

Reply:
done